docs/src/reference/the-traversal.asciidoc

```diff
@@ -4051,6 +4054,9 @@ g.V().valueMap().select('location').range(local, 1, 2).unfold() <3>
 <2> `List<String>` for each vertex containing the second location.
 <3> `String` for each vertex containing the second location (use `unfold()` to extract single elements from singleton collections).
 
+NOTE: When using globally-scoped `range()` inside <<repeat-step,`repeat()`>>, each repeat iteration maintains its own counter. 
```

Comment:
If I pretend I'm a new or casual Gremlin user I'm not sure that these `NOTE` callouts are helpful. I'm not sure I'd understand them and they seem to carry context that is implicit to this change that isn't clearly defined. Consider my comments at https://github.com/apache/tinkerpop/pull/3241/files#r2445906983 in deciding what happens here next I guess.